frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -605,8 +619,52 @@ function ReaderFooter:init()
     end
 
     self.visibility_change = nil
+
+    self.custom_text = G_reader_settings:readSetting("reader_footer_custom_text") or "KOReader"
 end
 
+function ReaderFooter:set_custom_text(touchmenu_instance)
+    local text_dialog
+    text_dialog = InputDialog:new{
+        title = _("Enter a custom text"),
+        description = _("A series of spaces can be entered as spacer placeholders."),
+        input = self.custom_text,
+        buttons = {
+            {
+                {
+                    text = _("Cancel"),
+                    callback = function()
+                        UIManager:close(text_dialog)
+                    end,
+                },
+                {
+                    text = _("OK"),
+                    callback = function()
+                        local new_text = text_dialog:getInputText()
+                        if self.custom_text ~= new_text then
+                            self.custom_text = new_text
+                            G_reader_settings:saveSetting("reader_footer_custom_text",
+                                self.custom_text)
+                            UIManager:close(text_dialog)
+                            self:refreshFooter(true, true)
+                            if touchmenu_instance then touchmenu_instance:updateItems() end
+                        end
+                    end,
+                },
+            }}
+        }
+    UIManager:show(text_dialog)
+    text_dialog:onShowKeyboard()
+end
+
+-- add functions or text here:
+-- Functions are executed and text will shown on a long press on a minibar option
+local minibar_option = {}
```

Comment:
```suggestion
-- Help text string, or function, to be shown, or executed, on a long press on menu item
local option_help_text = {}
```
(Because https://en.wikipedia.org/wiki/Minibar :)